.github/workflows/c-cpp.yml

```diff
@@ -5,32 +5,23 @@ on:
     branches: [ "yiyang" ]
   push:
     branches: [ "yiyang" ]
-
 jobs:
-  test-on-arm64:
+  test-wasm:
     runs-on: ubuntu-latest
     steps:
       - 
         name: install build essentials
         run: 
           sudo apt update && 
-          sudo apt install gcc g++ libgtest-dev cmake make build-essential sed gcc-aarch64-linux-gnu g++-aarch64-linux-gnu  
+          sudo apt install nodejs
       
-      - 
-        name: install qemu
-        run: 
-          sudo apt install qemu-user
-        
-      - name: cross-compile gtest for arm64
-        run: |
-          git clone https://github.com/google/googletest.git && 
-          mkdir gtest-build && 
-          cd gtest-build && cmake ../googletest/ -DCMAKE_INSTALL_PREFIX=/usr/aarch64-linux-gnu -DCMAKE_CXX_COMPILER=/usr/bin/aarch64-linux-gnu-g++ && 
-          make && sudo make install
```

Comment:
remain the old test and append the new, organize src and test files in your repo